src/components/Graphs/Graph.module.css

```
@@ -1,14 +1,43 @@
 .graphContainer {
   display: flex;
   height: 600px;
-  overflow: hidden;
+  padding: 2px; 
   font-family: var(--sapFontFamily);
+  position: relative;
 }
 
 .graphColumn {
   flex: 1;
   display: flex;
   flex-direction: column;
+  background-color: var(--sapTile_Background, #fff);
+}
+
+.topLegendContainer {
+  position: absolute;
+  top: 1rem;
+  right: 1rem;              
+  display: flex;
+  align-items: center;
+  
+  z-index: 2;
+}
+
+.filterIcon {
```

That's only for the container? What do you thing about giving it more precise name?
```suggestion
.filterIconContainer {
```